sdk/azidentity/client_certificate_credential_test.go

```diff
@@ -226,6 +227,33 @@ func TestClientCertificateCredential_Live(t *testing.T) {
 	}
 }
 
+func TestClientCertificateCredentialADFS_Live(t *testing.T) {
```

Comment:
I ended up creating a new test after considering it. There is a different tenant id, client id, authhost, etc. Essentially every env variable would be different from the other run. Also, ADFS cert authentication needed a password, which was not on the previous test. Additionally, I only checked PFX, as I created my graph application in the ADFS stack with a PFX cert. While I could've created a PEM file aswell, I believe that is unnecessary, due to the instanceDiscoveryFlag not affecting the flow of certificate parsing.